### DISCUSSION TITLE
Bug: adapter-qq 插件启动失败

### DISCUSSION THREAD

**Discussion by oppsDayly** [koishijs/koishi#1338] [Q&A]
### Describe the bug

window下qq官方机器人插件启动失败；链接wss时候报错invalid session，

### Steps to reproduce

window serve2012

### Expected behavior

正常启动

### Versions

Windows servers 2012

**Comment by ilharp** [comment]
使用问题请前往 Discussions 版或 Koishi 论坛讨论。已将此 Issue 转换为 Discussion。

请先查看这个帖子，提供列出的所有有效信息：<https://forum.koishi.xyz/t/topic/555>
